policyengine_canada/parameters/gov/provinces/mb/benefits/mbcb/reduction/three_children.yaml

```diff
@@ -0,0 +1,26 @@
+description: Manitoba phases the child benefit out for families with three children at this rate.
+brackets:
+  - threshold:
+      2022-01-01: 0
+    rate:
+      2022-01-01: 0
+  - threshold:
+      2022-01-01: 15_000
+    rate:
+      2022-01-01: 0.2318
+  - threshold:
+      2022-01-01: 20_435
+    rate:
+      2022-01-01: 1
+      
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-CAD
+  rate_unit: /1
+  label: Manibota Child Benefit three children phase out rate
+  reference:
+    - title: Manibota child tax benefit 
+      href: https://www.gov.mb.ca/fs/eia/mcb_table.html
+    - title: Manibota child tax benefit regulation
```

Comment:
state specific parts of the source where the reduction percentage can be found - in all reduction yaml files